third-party/gtest-1.8.1/fused-src/gtest/gtest.h

```diff
@@ -2442,6 +2443,10 @@ GTEST_API_ bool IsTrue(bool condition);
 
 // Defines scoped_ptr.
 
+// RocksDB: use unique_ptr to work around some clang-analyze false reports
+template <typename T>
+using scoped_ptr = std::unique_ptr<T>;
```

Comment:
Do we also need to comment out the entire class scoped_ptr for clarifity?

Reply:
It is commented out. For this third party code, I figured commenting out was better than removing (in case of future merge).

Reply:
oh - I missed those cute `/* */`. Thanks!